core/src/test/scala/org/apache/spark/util/UtilsSuite.scala

```diff
@@ -17,12 +17,13 @@
 
 package org.apache.spark.util
 
-import java.io.{ByteArrayInputStream, ByteArrayOutputStream, DataOutput, DataOutputStream, File,
-  FileOutputStream, PrintStream, SequenceInputStream}
+import java.io._
```

Comment:
Nit: should keep the original import list and just add the new class you are importing.

Reply:
Wu Yi recommended to change to java.io._ in former comments so I changed it.

Reply:
I don't remember I recommended ever...but yes it's recommended to use wildcard imports when [there're more than  6 entities](https://github.com/databricks/scala-style-guide#imports).

Reply:
You did recommend this change in former reviews. https://github.com/apache/spark/pull/32007#discussion_r622840432